### PR TITLE
committing log code for test and sending pull request

### PR DESCRIPTION
**Pull request by Senthamizha**
Testing to create pull request

### CODE REVIEW

**Comment by aravind06** [comment]
Please update logs in all service class

File: src/main/java/com/Amazon/Repository/CartRepository.java [hunk 1]
```diff
@@ -101,7 +101,7 @@ public CommonResponse itemsCount(String userName) {
 			Session session = getSession();
             int result= (int)((long)session.createQuery("select count(p) from CartItem p where p.userName = :userName")
```
Comment (aravind06) [inline src/main/java/com/Amazon/Repository/CartRepository.java:102]: why are we casting it to two different datatype here?

**Review by aravind06 — CHANGES_REQUESTED** [review]
Please update this and let me know

File: src/main/java/com/Amazon/Repository/CartRepository.java (hunk 1 above)
Comment (Senthamizha) [reply src/main/java/com/Amazon/Repository/CartRepository.java:102]: will update

**Review by aravind06 — APPROVED** [review]

**Pull request merged**